Launcher/Views/Sidebar/CLSidebarOptionMenuView.swift

```diff
@@ -19,7 +19,13 @@ struct CLSidebarOptionMenuView: View {
             } label: {
                 Text("Edit")
             }
-
+            #if DEBUG
```

Comment:
Nice feature! We can leave it for debug here, but this can be useful in everyday use. We'll remove the debug flag when we find it handy or get user feedback.

Reply:
Sure.